Tools/YamlCreate.ps1

```diff
@@ -2254,7 +2254,7 @@ Function Write-ManifestContent {
       "# yaml-language-server: `$schema=$Schema";
       '';
       # This regex looks for lines with the special character ⍰ and comments them out
-      $(ConvertTo-Yaml $YamlContent).TrimEnd() -replace "(.*)$([char]0x2370)", "# `$1"
+      $(ConvertTo-Yaml $YamlContent).TrimEnd() -replace "(.*) $([char]0x2370)", "# `$1"
```

Comment:
```suggestion
      $(ConvertTo-Yaml $YamlContent).TrimEnd() -replace "(.*)\s*$([char]0x2370)", "# `$1"
```
I know the script itself will only use a single standard space, but the regex should probably be written to replace any number of whitespace characters just as a precaution

Reply:
This does not work for some reasons 🤔

Reply:
> This does not work for some reasons 🤔

What if you try `\s+` instead? I'm guessing the space might be getting caught in the greedy capture of the `.*`